src/components/CrimCodeGridList.js

```diff
@@ -47,12 +47,13 @@ const CrimCodeGridList = props => {
     componentKey = props.heading1Key;
     componentLabel = props.heading1Label;
     componentTitle = props.heading1TitleText;
-  } else {    //SectionCCScreen.js
-    componentKey = props.headingKey;
+  } else {
```

Comment:
Eventually the else needs to be removed and make this condition equal to SectionCCScreen etc.

Reply:
I agree.